### DISCUSSION TITLE
Barter market place

### DISCUSSION THREAD

**Discussion by c-korifidis** [medusajs/medusa#9923] [Q&A]
Hello all and congrats for this project!

I have a request to build a multi vendor  bartering market place.
In other words a market place with no cash, where users can exchange goods with other user.
The don't buy each others product. 
They just say for example give me this chair and I 'll give you this bed.

Has someone made something similar so she/he can point me to the right plugins/guides etc?

Thank you!

**Comment by SalahAdDin** [comment]
> Hello all and congrats for this project!
> 
> I have a request to build a multi vendor bartering market place. In other words a market place with no cash, where users can exchange goods with other user. The don't buy each others product. They just say for example give me this chair and I 'll give you this bed.
> 
> Has someone made something similar so she/he can point me to the right plugins/guides etc?
> 
> Thank you!
There are to existing templates for multivendor shop, I don't remember the links.
